include/denormals.h

```diff
@@ -4,27 +4,36 @@
 #ifndef DENORMALS_H
 #define DENORMALS_H
 
+#include <immintrin.h>
+#include <fxsrintrin.h>
```

Comment:
This will break builds for non-x86 architectures. Also, including `fxsrintrin.h` directly is not required and not recommended.

Reply:
I think `fxsrintrin.h` was needed for getting the build working on Ubuntu 18.04, so I'm leaving it in for now.

Reply:
You're right, only `x86intrin.h` includes `fxsrintrin.h` in that case.

Reply:
I'm going with the recipe at https://newbedev.com/header-files-for-x86-simd-intrinsics, hope it covers all compilers we care about.